### DISCUSSION TITLE
TOML rule to replace expression with some value

### DISCUSSION THREAD

**Discussion by eshi13** [toml-lang/toml#1020] [Q&A]
I wrote this rule to replace `ccmContainer.stale_flag` with `treated`. But it doesn't seem to work. Can anyone help here?

```
[[rules]]
name = "test_rule_replace_true_placeholder"
query = """(
(navigation_expression
        target: (navigation_expression
            target: (simple_identifier)
            suffix: (navigation_suffix
                suffix: (simple_identifier) @access_identifier))
        suffix: (navigation_suffix
            suffix: (simple_identifier) @param)) @parameter_access
(#eq? @access_identifier "ccmContainer")
(#eq? @param "@stale_flag")
)"""
replace_node = "parameter_access"
replace = "@treated"
holes = ["stale_flag", "treated"]
groups = ["replace_expression_with_boolean_literal"]
```

**Comment by eksortso** [comment]
>  I wrote this rule to replace `ccmContainer.stale_flag` with `treated`. But it doesn't seem to work. Can anyone help here?

This is the repository for the TOML standard, the TOML language itself. If you're working with a particular tool that uses TOML and you're having issues with that tool, you will need to contact the project for that specific tool.
